tools/cli/pkg/command/completion.go

```diff
@@ -0,0 +1,118 @@
+package command
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+// CompletionCommand represents an execution of the kcp completion command
+type CompletionCommand struct {
+	cobraCmd       *cobra.Command
+	completeTarget string
+}
+
+var defaultFile string
+var validArguements = []string{"bash", "zsh", "fish", "powershell"}
+
+const errorMsg = "Error: accepts 1 arg(s), received 0 \n"
```

Comment:
Error: prefix is appended automatically by cobra when printing the message AFAIK

Reply:
Now removed the `Error:`.